src/main/java/de/doubleslash/keeptime/controller/Controller.java

```diff
@@ -142,6 +142,11 @@ public void deleteProject(final Project p) {
          LOG.error("You cannot delete the default project. Tried to delete project '{}'", p);
          return;
       }
+
+      if (p == model.activeWorkItem.get().getProject()) {
```

Comment:
not clear what this does. add a comment, but better a function like isProjectActive(p)